crates/types/src/data.rs

```diff
@@ -801,3 +803,46 @@ pub mod null_block {
         }
     }
 }
+
+/// A packed bundle constructed from a sequence of bundles.
+#[derive(Debug, Eq, PartialEq, Clone)]
+pub struct PackedBundle<TYPES: NodeType> {
```

Comment:
I really like the new type, though I wonder if there's a more descriptive name. `TransactionBundleData`?

I'm not really that familiar with the block/transaction side, so if you think `PackedBundle` is more appropriate I trust your judgment!